app/assets/javascripts/staff/emails.js

```diff
@@ -0,0 +1,37 @@
+$(document).ready(function() {
+  $('.template-exit-preview-btn, .template-preview, .template-edit, .email-markup-help').hide();
+  $('.template-save-btn, .template-remove-btn, .template-cancel-btn').hide();
+
+  $('.template-preview-btn').click(function(e) {
+    e.preventDefault();
+    $parent = $(this).parents('.template-section');
+    $parent.find('.template-short, .template-edit').hide();
+    $parent.find('.template-preview-btn').hide();
+    $parent.find('.template-preview').show();
+    $parent.find('.template-exit-preview-btn').show();
+    $('.email-markup-help').show();
```

Comment:
marty made the argument that we would like the help during preview. it could help people decide if they should edit.